types/index.d.ts

```diff
@@ -781,7 +781,11 @@ declare module "jspdf" {
       options?: { filename?: string }
     ): string;
     output(
-      type: "pdfobjectnewwindow" | "pdfjsnewwindow" | "dataurlnewwindow"
+      type: "pdfobjectnewwindow" | "pdfjsnewwindow"
+    ): Window;
+    output(
+      type: "dataurlnewwindow",
+      options?: { filename?: string }
```

Comment:
pdfobjectnewwindow and pdfjsnewwindow also accept the options object.

Reply:
@HackbrettXXX 
Thanks!
I fixed the options object.